<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_msg_dialog</class>
 <widget class="QDialog" name="admin_msg_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Warning</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../images/app-logo.jpg</normaloff>../images/app-logo.jpg</iconset>
  </property>
  <widget class="QLabel" name="background_img_label">
   <property name="geometry">
    <rect>
     <x>-190</x>
     <y>-20</y>
     <width>600</width>
     <height>500</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../images/background users.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="msg_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="accept_button">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decline_button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Decline</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
